### PR TITLE
fix: Properly calculate NavBar height based on viewport width

### PR DESCRIPTION
**Pull request by ChristianRomberg**
Please note that this does change the scrolling behavior. Please test before reviewing.

### CODE REVIEW

File: src/components/NavBar.tsx [hunk 1]
```diff
@@ -24,11 +24,12 @@ const Logo = config.ui?.Logo;
 
 const useStyles = makeStyles((theme) => ({
   appBar: {
-    position: "fixed",
+    position: "relative",
```
Comment (ChristophWersal) [inline src/components/NavBar.tsx:27]:
```suggestion
    position: "sticky",
```
This should have the same sticky behaviour as before (which I like), but without the need to add the fixed 64px padding on top.

**Review by ChristophWersal — COMMENTED** [review]
The Charts.tsx component also adds an explicit distance at the top. Maybe this should be removed as well. 
The only issue: on the mobile view the floating menu icon overlaps with the graph. So maybe some distance on top is needed if the user is on the mobile view? Or a title over the graph so that the distance is introduced more naturally?

**Comment by ChristianRomberg** [comment]
I think for the charts page a seperate issue should be opened if this turns out to be a problem.
I am in favor of a title for the charts page though.

**Review by ChristophWersal — APPROVED** [review]

**Pull request merged**